dev-app/src/screens/DiscoverReadersScreen.tsx

```diff
@@ -94,6 +94,9 @@ export default function DiscoverReadersScreen() {
     onDidReportAvailableUpdate: (update) => {
       Alert.alert('New update is available', update.deviceSoftwareVersion);
     },
+    onDidAcceptTermsOfService: () => {
+      Alert.alert('Accept terms of Service');
```

Comment:
nit: Should this be "Accepted Terms of Service"? 
not too important since it's just the dev app, but just fyi that this callback fires _after_ acceptance